<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>10</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="pushButton_Salvar">
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_putVoxel">
         <property name="text">
          <string>Put Voxel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cutVoxel">
         <property name="text">
          <string>Cut Voxel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SelecionaCor">
         <property name="text">
          <string>Selecionar Cor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="70,30">
       <item>
        <widget class="Plotter" name="widget" native="true"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Percorrer Matriz</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_Xsculptor">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_PosicaoX"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_Ysculptor">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_PosicaoY"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_Zsculptor">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_PosicaoZ"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Ret√¢ngulo</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QPushButton" name="pushButton_putRect">
                   <property name="text">
                    <string>Put Rect</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_cutRect">
                   <property name="text">
                    <string>Cut Rect</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>X</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider_Xret">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLCDNumber" name="lcdNumber_Xret"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>Y</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider_Yret">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLCDNumber" name="lcdNumber_Yret"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>Z</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider_Zret">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLCDNumber" name="lcdNumber_Zret"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QPushButton" name="pushButton_putSphere">
                 <property name="text">
                  <string>Put Sphere</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_cutSphere">
                 <property name="text">
                  <string>Cut Sphere</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Raio esfera</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QSlider" name="horizontalSlider_RaioEsfera">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_RaioEsfera"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Raio Elipsoide</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QPushButton" name="pushButton_putEllipsoid">
                 <property name="text">
                  <string>Put Ellipsoid</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_cutEllipsoid">
                 <property name="text">
                  <string>Cut Ellipsoid</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_Xellipsoid">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber_Xellipsoid"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_Yellipsoid">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber_Yellipsoid"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Z</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_Zellipsoid">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber_Zellipsoid"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSalvar_off"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSalvar_off">
   <property name="text">
    <string>Salvar .off</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Xellipsoid</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Xellipsoid</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Yellipsoid</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Yellipsoid</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Zellipsoid</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Zellipsoid</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RaioEsfera</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_RaioEsfera</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Zret</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Zret</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Yret</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Yret</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Xret</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_Xret</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Zsculptor</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_PosicaoZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Ysculptor</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_PosicaoY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Xsculptor</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_PosicaoX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
